libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -223,6 +223,10 @@ class AC_PrecLand
     void Write_Precland();
     uint32_t last_log_ms;  // last time we logged
 
+    bool option_enabled(Option option) const {
+        return (_options & uint8_t(option)) != 0;
```

Comment:
```suggestion
        return (_options & option:get()) != 0;
```

Reply:
I think we want do this here. `Option is an enumeration we just defined in AP_Precland. It doesn't have this `get()` method.

Reply:
Your right, get should be on the param. Should remove the need for casting.

Reply:
It's still needed. I have tried it. I have pushed the latest change on that line. 
Please clarify if I am misunderstanding something. What I did was, I changed
`return (_options & uint8_t(option)) != 0;` to `return (_options.get() & option) != 0;` but still there was need of casting `option` to uint8_t.